hack/jenkins/test-flake-chart/report_flakes/go.mod

```diff
@@ -1,6 +1,8 @@
 module flake
 
-go 1.22.3
```

Comment:
@einyx why do I see these diff ? do you mind rebasing your PR in a way that extra Diff doesnt show here